### PR TITLE
Update README.dev.md

### PR DESCRIPTION
**Pull request by abelsiqueira**
**Description**

Update README.dev.md

**Related issues**:
- #24 

**Instructions to review the pull request**

Review online.

### CODE REVIEW

File: README.dev.md [hunk 1]
```diff
@@ -3,9 +3,81 @@
...
+```
+
+Install [`npm`](https://www.npmjs.com).
+
+### GitHub token
+
+Go to [https://github.com/settings/tokens](https://github.com/settings/tokens) and create a GitHub token.
+You don't need to give it any permissions (we'll only read public information).
+We also recommend setting an expiry date.
+You only have you change to copy it, so don't miss it.
```
Comment (jspaaks) [inline README.dev.md:32]:
```suggestion
You only have one chance to copy it, so don't miss it.
```

**Review by jspaaks — APPROVED** [review]
Looks very nice! I see I was a bit premature with my earlier comment as the PR was still in Draft state. :1st_place_medal:

**Comment by abelsiqueira** [comment]
No _problemo_, no work lost.

**Comment by abelsiqueira** [comment]
Thanks for the review.

**Pull request merged**